includes/Feed/FeedFileHandler.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Feed;
+
+use Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class FeedFileHandler
+ *
+ * @since 2.6.0
+ */
+class FeedFileHandler {
+
+	const OPTION_FEED_URL_SECRET = 'wc_facebook_feed_url_secret';
+	const REQUEST_FEED_ACTION    = 'facebook_for_woocommerce_get_feed';
+
+	/**
+	 * Filename to export to.
+	 *
+	 * @var string
+	 */
+	public $filename = 'product_catalog.csv';
+
+	/**
+	 * Temporary export file.
+	 *
+	 * @var string
+	 */
+	public $temp_filename = 'temp_product_catalog.csv';
```

Comment:
I am adding the feed secret to the filename. This is how it was done before. I thought that we could avoid that but it will make the plugin work in Multisite configurations.

> or even use an OS temporary file (tmpfile or tmpnam).

We should not do this based on[ #1901 (comment)](https://github.com/woocommerce/facebook-for-woocommerce/issues/1901#issuecomment-837687668)

Reply:
Thanks for clarifying (we should get a review from @WPprodigy too!). 

I think it's worth adding a bit more docs/info about exactly how the file is generated, over multiple jobs - so the lifecycle of the temp file is super clear.